src/index.html

```diff
@@ -20,8 +21,348 @@
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__link-item">
+            <a
+              class="nav__link is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review_count">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price_text">Price:</p>
+          <p class="card__price_count">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
This 'card' block seems to be repeated multiple times. It would be better to extract this into a separate component and reuse it, to avoid code duplication. This is more of a suggestion for improving the code structure and readability.